src/RaceChrono.cpp

```diff
@@ -72,7 +130,34 @@ void RaceChronoBleAgent::setUp(
 
   Bluefruit.setTxPower(+4);
 }
+#elif defined(ARDUINO_ARCH_ESP32)
+void RaceChronoBleAgent::setUp(
+    const char *bluetoothName, RaceChronoBleCanHandler *handler) {
+  ::handler = handler;
+  BTName = bluetoothName;
+  BLEDevice::init(BTName);
+  BLEDevice::setSecurityIOCap(BLE_HS_IO_NO_INPUT_OUTPUT);
+  pServer = BLEDevice::createServer();
+  pMainService = pServer->createService(MAIN_SERVICE_NAME);
+
+  _pidRequestsCharacteristic = pMainService->createCharacteristic( 
```

Comment:
style preference / nit: take a look at how I break long lines that have an assignment.
Basically I break the line immediately after =, and that allows me to use more horizontal space on subsequent lines.
Also I use fewer spaces for "continuation indent".